src/App.js

```diff
@@ -1,15 +1,34 @@
 import React from 'react';
+const todoList = [
+  {
+    id: 1,
+    title: " Complete Assignment",
+   },
+   {
+     id: 2,
+     title: " Read",
+   },
+   {
+     id: 3,
+     title: " Be active",
+   },
+];
 
 function App() {
   return (
     <div style={{ textAlign: 'center' }}>
       <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
+        <h1>Todo List</h1>
+          <ul>
+            {todoList.map(function(item) {
+              return (
+                <li key={item.objectID}>
+                  <span>{item.id}</span>
```

Comment:
Great job with the mapping function!

You should be seeing an error on console like this:
![image](https://user-images.githubusercontent.com/8095975/225188512-74ffd7f4-dc38-4b85-900c-019fb7433630.png)

This happens because you are trying to pass `item.objectID` as a key, but the property `objectID` is not defined in your todoList object.

You can use the id in this case as the key like this and even reuse to show it in your list if you want:

```
<li key={item.id}>
  <span>{item.id}</span>
```